### ISSUE TITLE
GLIBC not found for Turing node on v2.0.2

### ISSUE THREAD

**Issue by jakobilobi** [OAK-Foundation/OAK-blockchain#475]
**Question**

Hi there,
I'm from Dwellir, a node provider running a node on the Turing chain, and when I just now tried upgrading our node from `v2.0.1` to `v2.0.2` I got this error when starting the node client (`oak-collator` client is renamed `polkadot` in our infra) after the upgrade:

```
/home/polkadot/polkadot: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /home/polkadot/polkadot)
/home/polkadot/polkadot: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/polkadot/polkadot)
/home/polkadot/polkadot: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/polkadot/polkadot)
/home/polkadot/polkadot: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /home/polkadot/polkadot)
/home/polkadot/polkadot: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /home/polkadot/polkadot)
```

I couldn't find any info in the [latest release notes](https://github.com/OAK-Foundation/OAK-blockchain/releases/tag/v2.0.2) about any new requirements for this version. My temporary solution was to downgrade to `v2.0.1` again, which lets the node run fine, but I do have questions:

- Is this new requirement an intended change?
- What should we install to make the `v2.0.2` client work?

Cheers,
Jakob

**Comment by chrisli30** [comment]
Hey Jakob,

Thank you for the reporting. Hmm let us take a look, but the 2.0.2 is a minor upgrade and shouldn’t include any node changes or upgrade requirement. We are testing 2.1.0 node upgrade in staging which is major and require another upgrade next week.

**Comment by jakobilobi** [comment]
Hi there,

Tested with the new binary, works great :+1: 

Minor thing but noticed the client still presents itself as v2.0.0, even though it's the v2.0.2 one, thought you'd might want to know!

```
./oak-collator --version
2.0.0-a2a9cc182f3
```

**Comment by chrisli30** [comment]
@jakobilobi yes that was a mistake. 🥲 We re-created the 2.1.0 Release last night, https://github.com/OAK-Foundation/OAK-blockchain/releases/tag/v2.1.0
Could you redo the upgrade and swith to that? Sorry about the inconvenience.

**Comment by jakobilobi** [comment]
Ok, upgrading to `2.1.0` :ok:
